docs/root/configuration/http/http_filters/mcp_filter.rst

```diff
@@ -10,9 +10,54 @@ Configuration
 
 Example configuration:
 
-.. code-block:: yaml
+.. literalinclude:: _include/mcp-filter.yaml
+    :language: yaml
+    :lines: 78-80
+    :lineno-start: 78
+    :linenos:
+    :caption: :download:`mcp-filter.yaml <_include/mcp-filter.yaml>`
 
-  http_filters:
-  - name: envoy.filters.http.mcp
-    typed_config:
-      "@type": type.googleapis.com/envoy.extensions.filters.http.mcp.v3.Mcp
+MCP awareness policy
+--------------------
+
+A new filter is introducing the MCP JSON_RPC parser, and we can get the internals to apply some RBAC rules based on the MCP attributes:
```

Comment:
im not entirely following this - what is the filter new to - Envoy, this config, or this example?

im guessing its new to envoy - but then we dont want to write about the history in the docs (ie not changelogs) - so we should remove the "story"

likewise im confused by "introducing" - is it the case that the MCP filter is or contains a JSON_RPC parser?

last bit - i think that is the important bit in terms of how it works - i think put that into a new paragraph and give some context/explanation about rel to RBAC - eg something like "RBAC can be used to control access and authentication to or for the MCP filter" (not sure this is correct but its what i understood this to be saying)